.devcontainer/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Use a base image with Ubuntu
+FROM mcr.microsoft.com/vscode/devcontainers/base:ubuntu
+
+# Install Java 8, curl, and Python (required for Google Cloud SDK)
+RUN apt-get update && \
+    apt-get install -y openjdk-8-jdk curl python3 python3-pip && \
```

Comment:
Starting in October we will no longer be deploying with Java 8. This should be changed to Java 11.